operator/apis/loki/v1/v1.go

```diff
@@ -81,6 +81,8 @@ var (
 	ErrReplicationFactorToZonesRatio = errors.New("replication factor is greater than the number of available zones")
 	// ErrReplicationSpecConflict when both the ReplicationSpec and depricated ReplicationFactor are used.
 	ErrReplicationSpecConflict = errors.New("replicationSpec and replicationFactor (deprecated) cannot be used at the same time")
+	// ErrIPv6InstanceAddrTypeNotAllowed when the default InstanceAddrType is used with enableIPv6.
+	ErrIPv6InstanceAddrTypeNotAllowed = errors.New("instanceAddrType `default` cannot be used with enableIPv6 at the same time")
```

Comment:
Don't like the backticks in the output, but I don't think we have quoted any other values so far:

```suggestion
	ErrIPv6InstanceAddrTypeNotAllowed = errors.New(`instanceAddrType "default" cannot be used with enableIPv6 at the same time`)
```